includes/cc-picker.js

```diff
@@ -0,0 +1,3 @@
+jQuery(document).ready(function($){
```

Comment:
This whole document needs to have proper spacing before and after parenthesese

Reply:
noted, will review